build/pr-pipeline.yml

```diff
@@ -0,0 +1,195 @@
+
+trigger: none
+
+variables:
+- template: build-variables.yml
+- template: pr-variables.yml
+
+stages:
+- stage: BuildUnitTests
+  displayName: 'Build and run unit tests'
+  jobs:
+  - job: Windows
+    pool:
+      vmImage: $(WindowsVmImage)
+    steps:
+    - template: build.yml
+
+  - job: Linux
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+    - template: build.yml
+      parameters:
+        packageArtifacts: false
+
+  - template: docker-build-all.yml
+    parameters: 
+      tag: pr$(system.pullRequest.pullRequestNumber)
+
+- stage: provisionEnvironment
+  displayName: Provision Environment
+  dependsOn: []
+  jobs:
+  - job: provision
+    steps:
+    - task: AzurePowerShell@4
+      displayName: Provision Resource Group
+      inputs:
+        azureSubscription: 'Microsoft Health Open Source Subscription'
```

Comment:
One left! `$(ConnectedServiceName)`

Reply:
I finally did what I should have done in the first place, a global search. This is the last one for certain!